frontend/src/features/projects/spl/forms/CloseOutFinancialsForm.tsx

```diff
@@ -16,7 +15,8 @@ const CloseOutFinancialsForm = (props: CloseOutFinancialsFormProps) => {
       <Form.Row>
         <Col>
           <ProjectNotes
-            field={`notes[${NoteTypes.LoanTerms}].note`}
```

Comment:
Using array positioning is an invalid implementation and will result in data corruption